### PR TITLE
Earthquake map

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds a map of earthquakes and a simple Earthquake page.

Screen Shot 2022-09-08 at 6 37 59 PM

### CODE REVIEW

File: static/js/components/EarthquakeMap.jsx [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    .join(" / ");
+}
+
+function earthquakeCanObserve(nestedEarthquake) {
+  let color = "#f9d71c";
+  if (nestedEarthquake.is_night_astronomical_at_least_one) {
+    color = "#0c1445";
+  }
+  return color;
+}
```
Comment (guynir42) [inline static/js/components/EarthquakeMap.jsx:47]: Do we care when it is night or day for earthquakes?

File: static/js/components/EarthquakesFilterForm.jsx [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+  const { handleSubmit, register, control, reset, getValues } = useForm();
+
+  const handleClickReset = () => {
+    reset();
+  };
+
+  return (
+    <Paper className={classes.paper} variant="outlined">
+      <div>
+        <h4> Filter Gcn Events By</h4>
```
Comment (guynir42) [inline static/js/components/EarthquakesFilterForm.jsx:120]: Should be Filter Earthquakes By?

**Review by guynir42 — APPROVED** [review]
Looks good. A lot more in this PR than just an Earthquake Map. 

two minor corrections below:

File: static/js/components/EarthquakeMap.jsx (hunk 1 above)
Comment (mcoughlin) [reply static/js/components/EarthquakeMap.jsx:47]: Good point, will check for retracted or not for now.

File: static/js/components/EarthquakesFilterForm.jsx (hunk 2 above)
Comment (mcoughlin) [reply static/js/components/EarthquakesFilterForm.jsx:120]: Oops.

**Pull request merged**